app/airdrop_missions.go

```diff
@@ -0,0 +1,33 @@
+package app
```

Comment:
Can we move this file to the `pkg/airdrop` folder? For we can avoid increasing the `app` package